File: 6/ssd.js
var readline = require('readline');
var rl = readline.createInterface({
	input:process.stdin,
	output:process.stdout
});


const sum = (a,c)=>a+c;
const sq = x=>x*x;


rl.on('line',input=>{
	console.log(parseInt(input)<=0||isNaN(parseInt(input))?0:
		sq(Array.from({length:parseInt(input)},(a,i)=>i+1)
		.reduce(sum))
		-
		Array.from({length:parseInt(input)},(a,i)=>i+1)
			.map(sq).reduce(sum)
	);
});

File: 6/ssd.hs

lsq :: [Int] -> Int
lsq [] = 0
lsq (x:xs) = x*x+lsq xs

suml :: [Int] -> Int
suml [] = 0
suml (x:xs) = x + sum xs

f :: Int -> Int
f n = (suml [1..n])*(suml [1..n])-(lsq [1..n])

main = do
 input <- getLine
 print (f (read input))
